### PR TITLE
[react-native] Support flashScrollIndicators on ScrollView

### PR DESCRIPTION
**Pull request by alpha0010**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test YOUR_PACKAGE_NAME`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [ ] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

I was unable to run tests/lint due to errors finding `csstype`. Uncertain what that is about.

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://reactnative.dev/docs/next/scrollview#flashscrollindicators
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.
- [ ] Include [tests for your changes](https://github.com/DefinitelyTyped/DefinitelyTyped#testing)
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`. If for reason the any rule need to be disabled, disable it for that line using `// tslint:disable-next-line [ruleName]` and not for whole package so that the need for disabling can be reviewed.

### CODE REVIEW

File: types/react-native/index.d.ts [hunk 1]
```diff
@@ -6885,6 +6885,11 @@ export class ScrollView extends ScrollViewBase {
      */
     scrollToEnd(options?: { animated: boolean }): void;
 
+    /**
+     * Displays the scroll indicators momentarily.
+     */
+    flashScrollIndicators(): void;
```
Comment (ca057) [inline types/react-native/index.d.ts:6891]: Shall we add a test for this?

**Comment by alloy** [comment]
> I was unable to run tests/lint due to errors finding csstype. Uncertain what that is about.

This is indeed a little unclear to me as well, but to locally fix it simply run `npm install csstype`

File: types/react-native/index.d.ts (hunk 1 above)
Comment (alloy) [reply types/react-native/index.d.ts:6891]: I think this is an example of what is fine not to test, it won’t add anything. I think tests are important to test the types that TS will enforce through the system, but this method takes no arguments nor does it return anything.

**Comment by alloy** [comment]
Thanks!

File: types/react-native/index.d.ts (hunk 1 above)
Comment (ca057) [reply types/react-native/index.d.ts:6891]: Good point, thanks for the clarification!

**Pull request merged**